demo/src/NeuralPolicy.js

```diff
@@ -1,3 +1,10 @@
+function sampleNormal(mean, stdDev) {
+  const u = Math.random();
+  const v = Math.random();
+  const z = Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v);
```

Comment:
we have log here, what if u is 0?

Reply:
`u = 1 - Math.random()` should fix it.

```
0 <= random() < 1
-1 < -random() <= 0
0 < (u = 1 - random()) <= 1
-∞ < log(u) <= 0
∞ > -2 log(u) >= 0
```

So `sqrt` will always get a non-negative value.